sshd-core/src/main/java/org/apache/sshd/client/config/keys/ClientIdentity.java

```diff
@@ -320,6 +321,10 @@ public static KeyPairProvider loadDefaultKeyPairProvider(
                 if (perm != null) {
                     continue;
                 }
+                String owner = KeyUtils.validateStrictKeyFileOwner(p, options);
+                if (owner != null) {
+                    continue;
+                }
             }
```

Comment:
I prefer:
```java
if (GenericUtils.length(owner) > 0) {
    continue;
}
```
I don't think "empty" owner are expected, and even if they are IMO they are the same as "no user".